examples/wasm_node/test/test.js

```diff
@@ -4,36 +4,36 @@ const bip39 = require('bip39');
 const getDigest = require('./utils').getDigest;
 const secp256k1 = require('secp256k1');
 const fs = require('fs');
-var assert = require('assert');
+const assert = require('assert');
 
 const EXAMPLE_MNEMONIC = "equip will roof matter pink blind book anxiety banner elbow sun young";
 
-const EXAMPLE_CBOR_TX = "885501fd1d0f4dfcd7e99afcb99a8326b7dc459d32c62855010f323f4709e8e4db0c1d4cd374f9f35201d26fb20144000186a0430009c4430061a80040";
+const EXAMPLE_CBOR_TX = "885501fd1d0f4dfcd7e99afcb99a8326b7dc459d32c62855010f323f4709e8e4db0c1d4cd374f9f35201d26fb20144000186a0430009c41961a80040";
 
 const EXAMPLE_TRANSACTION = {
     "to": "t17uoq6tp427uzv7fztkbsnn64iwotfrristwpryy",
     "from": "t1b4zd6ryj5dsnwda5jtjxj6ptkia5e35s52ox7ka",
     "nonce": 1,
     "value": "100000",
     "gasprice": "2500",
-    "gaslimit": "25000",
+    "gaslimit": 25000,
     "method": 0,
     "params": ""
 };
 
 const MASTER_KEY = "xprv9s21ZrQH143K49QgrAgAVELf6ue2tZNHYUc7yfj8JGZY9SpZ38u8EfhWi85GsA6grUeB36wXrbNTkjX9EfGP1ybbPRG4sdP2EPfY1SZ2BF5";
 let MASTER_NODE = bip32.fromBase58(MASTER_KEY);
 
-describe('Serialization / Deserialization', function() {
-    it('Valid cbor should be fine', function() {
+describe('Serialization / Deserialization', function () {
+    it('Valid cbor should be fine', function () {
         assert.strictEqual(JSON.stringify(EXAMPLE_TRANSACTION), signer_wasm.transaction_parse(EXAMPLE_CBOR_TX, true))
     });
 
-    it('Extra bytes should fail', function() {
+    it('Extra bytes should fail', function () {
         let cbor_transaction_extra_bytes = EXAMPLE_CBOR_TX + "00";
 
         assert.throws(
-            () => signer_wasm.transaction_parse(cbor_transaction_extra_bytes),
+            () => signer_wasm.transaction_parse(cbor_transaction_extra_bytes,),
```

Comment:
`transaction_parse` should take 2 parameters... The other parameter is testnet. Does it make it true by default ?

Reply:
good point.. the "advantages" of javascript. AFAIK, when something is missing, it will be `undefined`, in this case, WASM will later receive a false.
To avoid confusion, I have added an additional test case for this and made the previous test mode specific by checking the complete error message.